tests/flows/fulfillment/test_renewal.py

```
@@ -0,0 +1,708 @@
+        order_id=renewal_order["id"],
+        product_id="PRD-1111-1111",
+        authorization_id="authorization-id",
+        adobe_customer_id="customer-id",
+        renewal_payload=renewal_payload,
+    )
+
+
+def plan_entry(
+    subscription_id="renewing-sub-id",
+    offer_id="65304578CA01A12",
+    renew=True,  # ruff:ignore[boolean-default-value-positional-argument]
```

Ruff (0.16.1)

[warning] Invalid rule code in suppression: boolean-default-value-positional-argument

Enable `lint.preview` to use rule names

(RUF102)